ui/components/src/StatChart/calculateFontSize.ts

```diff
@@ -0,0 +1,63 @@
+// Copyright 2023 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { useChartsTheme } from '../context/ChartsThemeProvider';
+
+interface CalculateFontSize {
+  text: string;
+  fontWeight: number;
+  width: number;
+  height: number;
+  lineHeight?: number; // default is 1.2
```

Comment:
You can use `@default 1.2` to specify this if you put some typedoc above the `lineHeight`. I could see that being nice for folks using vscode and other tools that will show the typedoc on hover.